modules/log6x-logging-clo-cli-install.adoc

```diff
@@ -0,0 +1,163 @@
+// Module is included in the following assemblies:
+//
+// 
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-installing-logging-operator_{context}"]
+= Installing {clo} by using the CLI
+
+Install {clo} on your {product-title} cluster to collect and forward logs to a log store by using the {product-title} command-line interface (CLI).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You installed the {oc-first}.
+* You installed and configured {loki-op}.
+* You have created the `openshift-logging` namespace.
+
+.Procedure
+
+. Create an `OperatorGroup` object:
++
+.Example `OperatorGroup` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  upgradeStrategy: Default
+----
+<1> You must specify `openshift-logging` as the namespace.
+
+. Apply the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `Subscription` object for {clo}:
++
+.Example `Subscription` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  channel: stable-6.<y> # <2>
+  installPlanApproval: Automatic # <3>
+  name: cluster-logging
+  source: redhat-operators # <4>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify `openshift-logging` as the namespace.
+<2> Specify `stable-6.<y>` as the channel.
+<3> If the approval strategy in the subscription is set to `Automatic`, the update process initiates as soon as a new operator version is available in the selected channel. If the approval strategy is set to `Manual`, you must manually approve pending updates.
+<4> Specify `redhat-operators` as the value. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the `CatalogSource` object that you created when you configured Operator Lifecycle Manager (OLM).
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a service account to be used by the log collector:
++
+[source,terminal]
+----
+$ oc create sa logging-collector -n openshift-logging
+----
+
+. Assign the necessary permissions to the service account for the collector to be able to collect and forward logs. In this example, the collector is provided permissions to collect logs from both infrastructure and application logs.
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user logging-collector-logs-writer -z logging-collector -n openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z logging-collector -n openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z logging-collector -n openshift-logging
```

Comment:
We do not include multiple commands in a single code block. Suggest to divide it in a separate instruction+command format.

Reply:
Considering the timeline for the release would you min if i do this as a part of https://issues.redhat.com/browse/OBSDOCS-1747?